src/main/java/io/airlift/slice/XxHash64.java

```diff
@@ -69,8 +69,8 @@ public static long hash(long seed, Slice data, int offset, int length)
         hash += length;
 
         // round to the closest 32 byte boundary
-        // this is the point up to which {@see #updateBody} processed
-        int index = length & 0xFFFFFFE0;
```

Comment:
I think is the standard way to write it and is more clear (the JDK does it this way), but I can leave it.